### PR TITLE
Yahya/features/helm spike

### PR DESCRIPTION
**Pull request by yahya130**
With docs and a sample chart

### CODE REVIEW

File: docs/spikes/helm/guide.md.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Upgrade an app:
+helm upgrade [release] [chart]
+
+Roll back a release:
+helm rollback [release] [revision]
+
+Create a chart (repo):
+helm create [name]
+
```
Comment (keijoraamat) [inline docs/spikes/helm/guide.md.md:43]: Before packaging [helm lint](https://helm.sh/docs/helm/helm_lint/) might help catching bugs.

**Review by keijoraamat — APPROVED** [review]

**Review by tonistuulNG — APPROVED** [review]

File: docs/spikes/helm/guide.md.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Helm/Helm charts
+This document outlines a high-level overview of Helm charts and how Helm charts will be used to deploy our services to Kubernetes. This document assumes you would have basic understanding of how manifest files work when deploying a resource to a Kubernetes cluster. As part of this documentation, there has also been an example Helm chart repo created here {URL}.
+
+## What is Helm?
+Helm is a Kubernetes deployment tool for automating creation, packaging, configuration, and deployment of applications and services to Kubernetes clusters. Essentially, imagine Helm as a package manager for the Kubernetes system much like choco is for windows, and the packages we install are the apps and services we want to deploy onto the Kubernetes cluster.
```
Comment (martinkearn) [inline docs/spikes/helm/guide.md.md:5]:
```suggestion
Helm is a Kubernetes deployment tool for automating creation, packaging, configuration, and deployment of applications and services to Kubernetes clusters. Essentially, imagine Helm as a package manager for the Kubernetes system much like choco is for Windows, and the packages we install are the apps and services we want to deploy onto the Kubernetes cluster.
```

File: docs/spikes/helm/guide.md.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## What is a Helm chart?
+A Helm chart is a collection of files that describe a related set of Kubernetes resources. A single chart might be used to deploy something simple, like a memcached pod, or something complex, like a full web app stack with HTTP servers, databases, caches, and so on. Below describes the file structure of the chart and its contents.
+
+#This is the repo that contains your chart
```
Comment (martinkearn) [inline docs/spikes/helm/guide.md.md:13]: Of this is a code sample, encapsulate it in opening and closing ```

**Comment by yahya130** [comment]
Thanks @martinkearn @keijoraamat for your review I have updated the document

**Pull request merged**